sdk/src/contexts/cosmos/context.ts

```diff
@@ -473,64 +471,10 @@ export class CosmosContext<
     id: string,
     chain: ChainName | ChainId,
   ): Promise<ParsedMessage | ParsedRelayerMessage> {
-    const client = await this.getCosmWasmClient(chain);
```

Comment:
Is throwing here going to cause an issue in the resume transaction flow? I see that this gets called in `getRouteFromTx` in `operator.ts`

Reply:
`getRouteFromTx` checks whether the source chain is a gateway chain before trying to retrieve the message (see [here](https://github.com/wormhole-foundation/wormhole-connect/blob/development/wormhole-connect/src/routes/operator.ts#L65)), so this method should never be called. Maybe I can mention that on the error message.